### ISSUE TITLE
create your own framework: framework ArgumentResolver

### ISSUE THREAD

**Issue by flip111** [symfony/symfony-docs#9369]
on page https://symfony.com/doc/current/create_framework/event_dispatcher.html the `Framework` contructor

```php
public function __construct(EventDispatcher $dispatcher, UrlMatcherInterface $matcher, ControllerResolverInterface $controllerResolver, ArgumentResolverInterface $argumentResolver)
```

On the next page https://symfony.com/doc/current/create_framework/http_kernel_httpkernelinterface.html

```php
$framework = new Simplex\Framework($dispatcher, $matcher, $resolver);
```

The fourth argument argumentResolver is missing and no new code is given to update the `Framework` class accordingly

**Comment by javiereguiluz** [comment]
@flip111 nice catch! We changed this recently when documenting the ArgumentResolvers (https://github.com/symfony/symfony-docs/commit/11920e30dd5fafddc9edabc8fd1ef0749ecd55de#diff-67929c1fab6bc58dd269f01d1c4bbc9d)  Could you please submit a pull request to fix the missing argument in https://symfony.com/doc/current/create_framework/http_kernel_httpkernelinterface.html?

This small changes can be done in the browser directly. Click on "Edit this page", make the change and just like that you can create a pull request in GitHub.

**Comment by flip111** [comment]
I also added `// ..` to indicate that one should include previous written code (this "technique" is also used in other places in create your own framework).

**Comment by javiereguiluz** [comment]
Fixed by #9417.
